topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -161,6 +166,8 @@ Below we describe how you can do this, but it is not necessary for this tutorial
 > >     > the gray (preparing/queued) and yellow (running) states to become green (success):
 > >     >
 > >     > ![`UCSC Main on Human: knownGene` dataset is green](../../images/101_04.png)
+> >     >
+> >     > Now you need to login once again in your account.
```

Comment:
I believe this is only true on Chrome currently, should we make it "You might need to login to Galaxy again" orso? What do you think?

Reply:
Sure :)